io_scene_godot/structures.py

```diff
@@ -279,6 +296,9 @@ def add_elements(self, list_of_lists):
         for lis in list_of_lists:
             self.extend(lis)
 
+    def add_elements_flat(self, array):
```

Comment:
You can simple call `Array().extend()` to achieve this,  Array inherit list

Reply:
fixed